charts/matrix-stack/templates/ess-library/_credentials.tpl

```diff
@@ -9,7 +9,7 @@ SPDX-License-Identifier: AGPL-3.0-only
 {{- $root := .root -}}
 {{- with required "element-io.ess-library.value-from-values-path missing context" .context -}}
 {{- $path := . -}}
-{{- $navigatedToPart := merge (dict) (mustDeepCopy $root.Values) -}}
+{{- $navigatedToPart := mergeOverwrite (mustDeepCopy $root.Values) (dict) -}}
```

Comment:
Ensure what that was supposed to do @benbz

Reply:
IIRC `$root.Values` isn't a standard map so it couldn't be traversed. The merge into a `dict` meant that it did become a standard map